src/test/java/org/fundacionjala/coding/wilson/PlanetTwisted37Test.java

```diff
@@ -0,0 +1,23 @@
+package org.fundacionjala.coding.wilson;
+import java.util.Arrays;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * @author wiltrouble
+ */
+
+public class PlanetTwisted37Test {
+
+    private final Integer[] numbersTest1Exp = new Integer[]{1, 2, 7, 4, 5, 6, 3, 8, 9};
```

Comment:
please define this as local final variable inside test method.

Reply:
static?